lib/src/app.dart

```diff
@@ -127,18 +128,32 @@ class _AmiAppState extends State<AmiApp> {
         child: MaterialApp.router(
           routerConfig: _router,
           themeMode: ThemeMode.system,
-          theme: _createTheme(ThemeData.light(), lightColorScheme),
-          darkTheme: _createTheme(ThemeData.dark(), darkColorScheme),
+          theme: _createTheme(false),
+          darkTheme: _createTheme(true),
```

Comment:
I agree to not make our sample apps compatible with dark mode. But I wanted to make sure this isn't something we all agree on [so I asked in slack](https://customerio.slack.com/archives/C05C2HF5VB2/p1686942778983109).

Reply:
Interesting question, thanks for bringing this up 😄 Responded on slack.